### DISCUSSION TITLE
But why?

### DISCUSSION THREAD

**Discussion by ghost** [kovidgoyal/kitty#6731] [General]
Hallo. I do not want to infuriate you but recently I stumbled upon your project kitty and I wondered why somebody takes the hardness and develops a GPU based terminal emulator. Because I personally had rarely ever in my whole life a situation in which I desired a faster or more streamlined terminal emulator. I use konsole on my private computer which never exceeds 1.3% of load and X usually between 1 and 6%. Which is quite livable and leaves much, much load capacity for other processes. May I ask you how (in what situation) you got these 29-31% with konsole? I mean I could dump a long file in a console window but it does not make much sense, does it? By shedding the load onto the GPU trades one category of problems against the other as you might have observed while writing your FAQ. Some of the servers I administrate don’t even have decent GPUs to speak of, or the drivers are buggy and prone to crash every moment, namely drivers for Intel boards and on-board graphics. You wouldn't want to administrate your server with a tool relying on weak drivers.

**Comment by kovidgoyal** [comment]
Read kitty https://sw.kovidgoyal.net/kitty/performance/ it describes
exactly what the circumstances are. And the absolute numbers are
irrelevant since it depends on your machine, the importance is the
*difference* between them.

And the GPUs on servers are irrelevant, since you ssh into servers you
dont run kitty on them.

And hey if you like burning excess energy and lowering the battery life
of your machines, good for you, stick with whatever you want. No one is
forcing you to use kitty.

Although I have to say, at this point performance is the least
compelling reason to use kitty, its design and features that allow you
to do things no other terminal does are.
